apps/web/src/components/UpsellBanner/UpsellBanner.tsx

```diff
@@ -0,0 +1,106 @@
+import { useCallback } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import IconContainer from '~/components/core/IconContainer';
+import { UpsellBannerQuery$key } from '~/generated/UpsellBannerQuery.graphql';
+import useAddWalletModal from '~/hooks/useAddWalletModal';
+import CloseIcon from '~/icons/CloseIcon';
+import colors from '~/shared/theme/colors';
+import useExperience from '~/utils/graphql/experiences/useExperience';
+
+import breakpoints from '../core/breakpoints';
+import { Button } from '../core/Button/Button';
+import { HStack, VStack } from '../core/Spacer/Stack';
+import { BaseM } from '../core/Text/Text';
+
+type Props = {
+  queryRef: UpsellBannerQuery$key;
+};
+
+const USER_EXPERIENCE_KEY = 'UpsellBanner';
```

Comment:
Need to wait for backend PR to merged. For testing, we could use different user experience key